products/views.py

```diff
@@ -1,3 +1,23 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http       import JsonResponse
+from django.views      import View
+
+from products.models   import Product, ProductImage
+class DetailView(View):
```

Comment:
항상 네이밍은 명확하게!
만약 다른 상세 정보 호출하는 view가 필요하다면 그때가서 모든 코드를 수정해야 합니다.
```suggestion
class ProductDetailView(View):
```